website/benchmarks.html

```diff
@@ -18,6 +18,8 @@ <h1>Deno Continuous Benchmarks</h1>
         <a href="https://github.com/denoland/deno">master branch</a>.
       </p>
 
+      <p style="font-size: 14px;font-style: italic;">Make sure your adblocker is disabled. Some blocks the plot rendering.</p>
```

Comment:
block => block.

Alternatively:

Make sure your adblocker is disabled as some can block the chart rendering.

Reply:
Also remove the styling.